### DISCUSSION TITLE
新手入站水贴

### DISCUSSION THREAD

**Discussion by Tangmumao9527** [share121/inter-knot#4832] [灌水]
你好（lbw音）
![Uploading Screenshot_20240618_134344.jpg…]()

**Comment by Robin-txdy** [comment]
泥豪<img src="https://img-static.mihoyo.com/communityweb/upload/a370edf80a844b3abcececf055ab9cf9.png" alt="三月七_盯" width="75" >

**Comment by swyh0648** [comment]
怎么发的啊，而且英文有点看不懂

**Reply by Tangmumao9527** [reply]
用浏览器的翻译，大概能看懂就行啦
